Source/View Controller/SponsorsTableViewController.swift

```diff
@@ -66,6 +66,22 @@ internal class SponsorsTableViewController: NavigationBarTableViewController, Na
         return cell
     }
     
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let destinationID = "MapViewController"
+        
+        guard let mapViewController = self.storyboard?.instantiateViewController(withIdentifier: destinationID) as? MapViewController
```

Comment:
DestinationID should be a static variable in `MapViewController` to be it easier to edit in the future. If it's defined at multiple places, it would be easy to miss one of them later.

Reply:
Also sponsors doesn't have any map feature. Its only for schedule and workshops.

Reply:
I think I misunderstood initially. I initially thought that having a static variable to denote the identifier would be harder change later on. I'll make sure to implement this after I fix the build failure due to the merge.